scripts/install-sysdig.in

```diff
@@ -144,8 +144,11 @@ if [ -f /etc/debian_version ]; then
 	esac
 
 elif [ -f /etc/system-release-cpe ]; then
-	DISTRO=$(cat /etc/system-release-cpe | cut -d':' -f3)
-	VERSION=$(cat /etc/system-release-cpe | cut -d':' -f5 | cut -d'.' -f1 | sed 's/[^0-9]*//g')
+	IFS=':' read -ra RELEASE <<< $(</etc/system-release-cpe)
```

Comment:
I quote this: I admit I am not a bash guru so this sounds much more tricky than the original "cut" approach, ideally I would like to see something that simplifies the code :)

But thanks a lot for working on this!